examples/blink.rs

```diff
@@ -7,10 +7,22 @@ use tock::timer;
 use tock::timer::Duration;
 
 fn main() {
-    let led = led::get(0).unwrap();
+    let num_leds = led::count();
 
+    // Blink the LEDs in a binary count pattern and scale
+    // to the number of LEDs on the board.
+    let mut count: usize = 0;
     loop {
-        led.toggle();
-        timer::sleep(Duration::from_ms(500));
+        for i in 0..num_leds {
+            if count & (1 << i) == (1 << i) {
+                led::get(i).unwrap().on();
+            } else {
+                led::get(i).unwrap().off();
+            }
+        }
+        count += 1;
```

Comment:
I would compute mod num_leds here.

Reply:
right, this will eventually overflow and panic...

Reply:
use ```checked_add```?

Reply:
@niklasad1 `wrapping_add` is what we want, I think. We always want it to wrap